src/lib/recurringPurposes.js

```diff
@@ -576,11 +577,21 @@ const findRecurringTripsFromTimeserie = async (
   return timeseriesToUpdate
 }
 
-const saveTrips = async (client, timeseriesToUpdate) => {
+const saveTrips = async ({ client, timeseriesToUpdate, t }) => {
   if (timeseriesToUpdate.length > 0) {
     log('info', `${timeseriesToUpdate.length} trips to update`)
+
+    for (const timeserie of timeseriesToUpdate) {
+      set(
+        timeserie,
+        'aggregation.automaticTitle',
+        makeAggregationTitle(timeserie, t)
+      )
+    }
+
     return client.saveAll(timeseriesToUpdate)
   }
+
```

Comment:
Let's keep it like this for now, but we should definitely move this method in timeseries.js, as it should be called anytime we need to save multiple trips